### PR TITLE
fix: try to improve transition on safari

### PR DESCRIPTION
**Pull request by Dank-del**

### CODE REVIEW

**Comment by irffanasiff** [comment]
@Dank-del it actually removed the transition from mac/web browser and now there is a different transition on mobile i dont think this is the solution to it - https://square-docs-git-transition-fix-cubik-team.vercel.app/components/disclosure/tabs

File: packages/ui/components/ui/Tabs/tabs.tsx
[the remark this answers is not in the document]
Comment (irffanasiff) [reply packages/ui/components/ui/Tabs/tabs.tsx:1]: @dhruvsol @Dank-del

**Review by irffanasiff — CHANGES_REQUESTED** [review]

**Comment by Dank-del** [comment]
unable to find the root cause of the issue but i will list my findings
- Using -webkit-transform: translate3d(0,0,0); to give a hardware acceleration did not work on safari
- rewrote the hook and memoized function sped up the ui a bit but did not completely fix the stuttery ui on safari
- [setting transition property to color also does not have any effect](https://joelglovier.com/writing/solution-for-css-transitions-crashing-safari)

**Comment by dhruvsol** [comment]
we solved it closing it

**Pull request closed without merging**